Userland/Applications/3DFileViewer/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ serenity_component(
     3DFileViewer
     RECOMMENDED
     TARGETS 3DFileViewer
+    DEPENDS FileSystemAccessServer
```

Comment:
Is this needed? I'm not too hot on CMake, but I don't think the other applications have this.

Reply:
I believe this is required for the component system to work.